blit.lisp

```diff
@@ -215,25 +215,39 @@ This variable is sometimes used to disable recording for editing purposes.")
 ;;;    **
 (defmethod blit (canvas (box row-box) x y)
   (let ((padding (row-box-padding box))
-        #+#:ignore
         (height (height box))
-        #+#:ignore
-        (baseline (baseline box))
+	(align (row-box-align box))
         (extent (height-above-baseline box)))
     (labels ((rec (boxes x)
                (unless (null boxes)
-                 (let ((the-box (first boxes)))
-                   (blit canvas
-                         the-box
-                         x
-                         ;; Align on the baselines.
-                         (+ y (- extent (height-above-baseline the-box)))
-                         #+#:ignore     ; This will vertically center
-                         (+ y (floor (- height (height the-box)) 2))))
-
-                 (rec (rest boxes) (+ x padding (width (first boxes)))))))
+                 (let* ((the-box (first boxes))
+			(delta
+			  (case align
```

Comment:
use `ecase` so we error on an invalid option. Change these to keywords (see my comment elsewhere).